gui/characterEditor.py

```diff
@@ -340,7 +340,7 @@ def __init__(self, parent):
         self.skillBookDirtyImageId = self.imageList.Add(wx.Icon(BitmapLoader.getBitmap("skill_small_red", "gui")))
 
         tree.AppendColumn(_("Skill"))
-        tree.AppendColumn(_("Level"), align=wx.ALIGN_CENTER)
+        tree.AppendColumn(_("Level"))
         # tree.SetMainColumn(0)
```

Comment:
curious why we got rid of the align?

Reply:
I think before this change "Level" is not aligned like "Skill":

before:
<img width="877" alt="Screen Shot 2020-06-21 at 12 37 58 AM" src="https://user-images.githubusercontent.com/6920098/85206891-c95c0880-b357-11ea-8135-3163b1d19546.png">

after:
<img width="964" alt="Screen Shot 2020-06-21 at 12 37 29 AM" src="https://user-images.githubusercontent.com/6920098/85206899-d24cda00-b357-11ea-9fda-18c7c700f79b.png">